README.md

```diff
@@ -1,8 +1,11 @@
 # SatML
+
```

Comment:
Can you remove these additional added lines so the diff is clean?

Reply:
i deleted the newlines on my end. when i saved files on vsc the prettier extension would add newlines automatically. i dont think i can send a second pull request with the deleted newlines

Reply:
Kk. Not a big deal.
If this becomes a problem when working with code files, you can probably use a `.prettierignore` file or disable format on save in vac for this project